src/App.jsx

```diff
@@ -18,13 +18,13 @@ function App() {
           <Nav.Link as={Link} to="/articles">Articles</Nav.Link>
         </Nav.Item>
       </Nav>
-      <div className='app-container'>
+      <main className='app-container'>
         <Routes>
-          <Route path="/articles" element={<DisplayArticles apiUrl={apiUrl} />} />
+          <Route path="/articles/*" element={<DisplayArticles />} />
```

Comment:
You may find it more useful here to add a parametric endpoint for article_id so that you can bring that value down from the url on a logical variable name. Later you could have more endpoints that come off of articles so it makes sense to be more specific if you can.